README.md

```diff
@@ -3025,7 +3025,7 @@ Other **table constraints** can involve several columns:
     t.uniqueKey(["a", "b"], onConfict: .replace)
     
     // FOREIGN KEY (a, b) REFERENCES parents(c, d),
-    t.foreignKey(["a", "b"], references: "parent")
+    t.foreignKey(["a", "b"], references: "parents")
```

Comment:
Not sure about this one. Doesn't
```swift
t.foreignKey(["a", "b"], references: "parents")
```
generate
```sql
FOREIGN KEY (a, b) REFERENCES parents(a, b)
```
?

Reply:
`t.foreignKey(["a", "b"], references: "parents")` generates `FOREIGN KEY (a, b) REFERENCES parents(c, d)` if the primary key of parents is (c, d).

I agree that this is not obvious.

You'll read above:

> When defining a foreign key, the referenced column is the primary key of the referenced table (unless you specify otherwise):
> 
> ```swift
> // countryCode TEXT REFERENCES countries(code) ON DELETE CASCADE,
> t.column("countryCode", .text).references("countries", onDelete: .cascade)
> ```

The goal is the same: you don't have to provide the primary key of the referenced table.